src/tests-bns/event-server-tests.ts

```diff
@@ -1027,6 +1027,56 @@ describe('BNS event server tests', () => {
     expect(namespaceList.results).toStrictEqual(['ape.mega']);
   });
 
+  test('IBD mode blocks certain API routes', async () => {
```

Comment:
These tests look great but they should go in the event replay test suite instead of BNS.

Reply:
done